temporian/core/operators/all_operators.py

```diff
@@ -3,7 +3,7 @@
 # pylint: disable=unused-import
 from temporian.core.operators.arithmetic import divide
 from temporian.core.operators.arithmetic import multiply
-from temporian.core.operators.arithmetic import substract
+from temporian.core.operators.arithmetic import subtract
 from temporian.core.operators.arithmetic import sum

```

Comment:
Maybe this is the right time to rename "sum" into "add" or "addition" (like the operator name).

Reply:
+1, otherwise we're overwriting a python built-in function

Reply:
Yeah, if there's no good reason to use `sum` I think it's better too. I'm on it.